packages/react-scripts/utils/createJestConfig.js

```diff
@@ -12,11 +12,18 @@
 const fs = require('fs');
 const paths = require('../config/paths');
 
-module.exports = (resolve, rootDir, isEjecting) => {
+module.exports = (opts) => {
+  const resolve = opts.resolve;
+  const rootDir = opts.rootDir || null;
+  const isEjecting = opts.isEjecting || false;
+  const isDebug = opts.isDebug || false;
+
   // Use this instead of `paths.testsSetup` to avoid putting
   // an absolute filename into configuration after ejecting.
   const setupTestsFile = fs.existsSync(paths.testsSetup) ? '<rootDir>/src/setupTests.js' : undefined;
-
+  const babelTransform = isEjecting
+    ? '<rootDir>/node_modules/babel-jest'
+    : resolve('config/jest/babelTransform' + (isDebug ? 'Debug' : '') + '.js')
```

Comment:
@Timer, :eyes:, this is where/why there were two babelTransform files.  we are passing the babel config as a file, therefore, to enable sourceMaps, i added a different file.  if there's another way to get those babel settings passed to jest, we could pursue that.  we could also set something in the ENV to drop it down to one transform file?